src/components/AppProvider/AppProvider.tsx

```diff
@@ -1,8 +1,16 @@
-import React from 'react';
+import React, { Suspense } from 'react';
+import { FirebaseAppProvider } from 'reactfire';
 import { ThemeDecorator } from 'utils/theme';
+import { firebaseConfig } from 'utils/firebase';
 
 const AppProvider: React.FC = ({ children }) => {
-  return <ThemeDecorator>{children}</ThemeDecorator>;
+  return (
+    <ThemeDecorator>
+      <FirebaseAppProvider firebaseConfig={firebaseConfig}>
```

Comment:
Why is the `ThemeDecorator` wrapping the `FirebaseAppProvider`. One day we might want to use something stored in firebase to determine which theme values to use. I think the `FirebaseAppProvider` should wrap the `ThemeDecorator` instead.

Reply:
Got it. Makes sense👍